utils/products.js

```diff
@@ -1,152 +1,676 @@
+    committers: [
+      "https://github.com/evegufy",
+      "https://github.com/stephanbcbauer",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request charts Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "data-exchange-test-service",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/data-exchange-test-service",
+    ],
+    committers: [
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request data-exchange-test-service Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "demand-capacity-mgmt",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/demand-capacity-mgmt"],
+    committers: ["https://github.com/nitin-vavdiya"],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request demand-capacity-mgmt Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "digital-product-pass",
+    productDescription: "digital product pass",
+    githubRepo: ["https://github.com/eclipse-tractusx/digital-product-pass"],
+    committers: [
+      "https://github.com/matbmoser",
+      "https://github.com/saudkhan116",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request digital-product-pass Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "eclipse-tractusx.github.io",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/eclipse-tractusx.github.io",
+    ],
+    committers: [
+      "https://github.com/stephanbcbauer",
+      "https://github.com/arnoweiss",
+      "https://github.com/maximilianong",
+      "https://github.com/mhellmeier",
+      "https://github.com/matbmoser",
+      "https://github.com/danielmiehle",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request eclipse-tractusx.github.io Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "eclipse-tractusx.github.io.largefiles",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/eclipse-tractusx.github.io.largefiles",
+    ],
+    committers: [
+      "https://github.com/matbmoser",
+      "https://github.com/stephanbcbauer",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request eclipse-tractusx.github.io.largefiles Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "eco-pass-kit",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/eco-pass-kit"],
+    committers: [
+      "https://github.com/matbmoser",
+      "https://github.com/saudkhan116",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request eco-pass-kit Team",
+    hasBoard: false,
+    showVersion: true,
+  },
```

Comment:
I will delete this product, this repo is archived and was moved to the main webpage of tractus-x